pom.xml

```diff
@@ -118,4 +118,17 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.sonarsource.sonar-packaging-maven-plugin</groupId>
+        <artifactId>sonar-packaging-maven-plugin</artifactId>
+        <configuration>
+          <pluginClass>org.sonarsource.auth.github.AuthGitHubPlugin</pluginClass>
+          <sonarQubeMinVersion>6.7</sonarQubeMinVersion>
```

Comment:
Does that mean that the currently released version does not support SQ 6.7 ? In that case it would deserve a dedicated ticket.

Reply:
Good point => https://jira.sonarsource.com/browse/SQAUTHGITH-22